redisinsight/ui/src/slices/rdi/pipeline.ts

```diff
@@ -119,6 +149,60 @@ export function deployPipelineAction(
   }
 }
 
+export function fetchPipelineStrategies(
+  rdiInstanceId: string,
+  // TODO update after confirm response with RDI team
+  onSuccessAction?: (data: unknown) => void,
+  onFailAction?: () => void,
+) {
+  return async (dispatch: AppDispatch) => {
+    try {
+      dispatch(getPipelineStrategies())
+      const { status, data } = await apiService.get(
+        getRdiUrl(rdiInstanceId, ApiEndpoints.RDI_PIPELINE_STRATEGIES),
+      )
+
+      if (isStatusSuccessful(status)) {
+        dispatch(getPipelineStrategiesSuccess(data))
+        onSuccessAction?.(data)
+      }
+    } catch (_err) {
+      const error = _err as AxiosError
+      const parsedError = getApiErrorMessage(error as EnhancedAxiosError)
+
+      dispatch(getPipelineStrategiesFailure(parsedError))
+      onFailAction?.()
+    }
+  }
+}
+
+export function fetchPipelineTemplate(
+  rdiInstanceId: string,
+  // TODO add interface
```

Comment:
Is this going to be completed in a separate card?

Reply:
Yep. the same. In rdi docs right now we have template for job and for config in one request. But in our realization it doesn't make sense.
I will take a look at the bug. Which you found